client/signup/steps/woocommerce-install/components/support-card/index.tsx

```diff
@@ -20,7 +20,7 @@ const SupportLinkStyle = styled.a`
 	font-weight: bold;
 `;
 
-export default function SupportCard(): ReactElement {
+export default function SupportCard( { backUrl }: { backUrl?: string } ): ReactElement {
```

Comment:
I'm missing why we need an object here 🤔 We're passing in a template literal, would `( backUrl: string )` work?

Reply:
I think is because this is a prop object